src/app/statistic/statistic.component.ts

```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'sh-statistic',
```

Comment:
What is difference between this component and already created 'sensor-statistic' that located in dashboard module?